unicore/comments/service/views/comments.py

```diff
@@ -86,5 +102,54 @@ def delete_comment(request, uuid, connection):
 
 
 '''
-TODO: Comment collection resource
+Comment collection resource
 '''
+
+
+@app.route('/comments/', methods=['GET'])
+@inlineCallbacks
+def list_comments(request):
+    columns = Comment.__table__.c
+    filter_expr = comment_filters.get_filter_expression(request.args, columns)
```

Comment:
Ah I see. `comment_filters` is actually a `colander.MappingSchema` custom subclass instance. It deserializes and validates the args before generating the expressions. Ideally I'd like us to be able to use it in a similar way to [django-tastypie filtering](https://django-tastypie.readthedocs.org/en/latest/resources.html#basic-filtering) where you just provide a dict of `field: [filters...]` pairs.